telemetry/event_counters.go

```diff
@@ -158,7 +163,11 @@ func RelayMiningDifficultyGauge(numbLeadingZeroBits int, serviceId string) {
 
 // RelayEMAGauge sets a gauge which tracks the relay EMA for a service.
 // The serviceId is used as a label to be able to track the EMA for each service.
-func RelayEMAGauge(relayEMA uint64, serviceId string) {
+func RelayEMAGauge(relayEMA uint64, serviceId string, err error) {
+	if err != nil {
```

Comment:
Wdyt about capturing the error without updating the gague (is this possible)? See [`ProofRequirementCounter`](https://github.com/pokt-network/poktroll/blob/main/telemetry/event_counters.go#L70).